### PR TITLE
Update FormToggle readme

### PR DESCRIPTION
**Pull request by melchoyce**
Updating documentation to describe the use and functionality of the FormToggle component.

Thanks to @drw158 and @sarahmonster for drafting this.

### CODE REVIEW

File: packages/components/src/form-toggle/README.md [hunk 1]
```diff
@@ -1,22 +1,56 @@
...
 
-The Form Toggle Control is a switch that should be **used when the effect is boolean and instant**. The Form Toggle Control is a complement to the Checkbox Control.
+![On and off FormToggles](https://wordpress.org/gutenberg/files/2019/01/Toggle.jpg)
 
-Use Form Toggle when:
+## Table of contents
 
-- The effect of the switch is immediately visible to the user. For example: when applying a drop-cap to text, the actual drop-cap is immediately activated.
-- There are only two states for the switch: on or off.
+1. [Design guidelines](http://#design-guidelines)
```
Comment (kjellr) [inline packages/components/src/form-toggle/README.md]: Looks like these anchor links have an unnecessary `http://` in them.

Comment (melchoyce) [reply packages/components/src/form-toggle/README.md]: Good catch 👍

**Review by kjellr — APPROVED** [review]
Added a small optional note. Otherwise, this should be ready to 🚢!

File: packages/components/src/form-toggle/README.md [hunk 2]
```diff
@@ -1,22 +1,56 @@
...
+**Don’t**
+Don’t use radio buttons for settings that toggle on and off.
+
+Toggles are preferred when the user is not expecting to submit data, as is the case with checkboxes and radio buttons.
+
+#### State
+
+When the user slides a toggle thumb (1) to the other side of the track (2) and the state of the toggle changes, it’s been successfully toggled.
+
+![Diagram showing FormToggle states](https://wordpress.org/gutenberg/files/2019/01/Toggle-Diagram.jpg)
```
[the remark this answers is not in the document]
Comment (melchoyce) [reply packages/components/src/form-toggle/README.md]: Yeah, I think that's a good idea — will update 👍

File: packages/components/src/form-toggle/README.md [hunk 3]
```diff
@@ -1,22 +1,56 @@
 # FormToggle
+FormToggle switches a single setting on or off.
```
Comment (chrisvanpatten) [inline packages/components/src/form-toggle/README.md]: Newline above here (our parser might handle it weirdly without the newline)

**Review by chrisvanpatten — APPROVED** [review]
One small formatting thing but overall looks great! Excited to see component docs coming together.

**Pull request merged**